cmd/juju/controller/destroy.go

```diff
@@ -119,6 +120,12 @@ WARNING! This command will destroy the %q controller.
 This includes all machines, applications, data and other resources.
 `[1:]
 
+var destroySysMsgWithDetails = `
+WARNING! This command will destroy the %q controller with %d model(s).
+The following models will be destroyed: %s.
+This includes all machines, applications, data and other resources.
+`[1:]
```

Comment:
This is _fine_. I would prefer a little more detail, however. You have the numebr of machines, volumes and filesystems already available. 

It should also be possible to get their names without too much work. It looks like (from a quick glance) this information is available in `newData`, but reduced down to just a count

Reply:
We can add app info after we modify the model status request. I think that will be in a separate PR.